### DISCUSSION TITLE
Newbie question: non-empty SITEURL resulting in web page rendering going out of whack

### DISCUSSION THREAD

**Discussion by avibathula** [getpelican/pelican#3282] [Q&A]
Hi - I just started learning Pelican and planning to use it to build my personal website. I installed the latest bits.

I am using attilla theme and the default pelicanconf.py sets 
`SITEURL=""`

With default value - my bare bones site is working fine 

![Web capture_30-1-2024_123515_127 0 0 1](https://github.com/getpelican/pelican/assets/5514875/e871266e-4f72-4bbe-9001-70e6ce60698a)

But as SITEURL is needed for feeds to get created properly, I changed it to 
 `SITEURL="https://blog.mysite.me" `

This change was enough to make the site render like below - What am I missing here? I verified that by unsetting the string, everything starts working again.

![Web capture_30-1-2024_123614_127 0 0 1](https://github.com/getpelican/pelican/assets/5514875/71c03c12-f0b4-40fb-abdb-a64d615ef717)

Regards,
    -Avi

**Comment by justinmayer** [comment]
Use your browser's developer tools to view your console, and you will probably see that your stylesheet is not loading properly. That is expected behavior because in the latter case your stylesheet contains the full absolute URL path instead of the _relative_ URL path.

Leave `SITEURL` as an empty string when developing locally, and only set it to your actual site domain in your _production_ settings. This is how things should already be set up when using the Pelican quick-start script.

**Comment by Epy** [comment]
Hi
You can also use the setting: `RELATIVE_URLS = True` while developping in _pelicanconf.py_, and overriding it in _publishconf.py_ file used for publishing online
